### PR TITLE
Update readme a1 to implement 1

### LINKED ISSUE #1: There should be a way to make a feature request more easily
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when it takes a lot of time to make a  feature request.

**Describe the solution you'd like**
It would be a bliss to have a trimmed down feature request.

**Additional context**
Ref: https://github.com/erebos-xyz/test/blob/master/.github/ISSUE_TEMPLATE/quick_request.md

### PR DESCRIPTION
**Pull request by kleberbaum**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [ ] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no way to make a quick feature request. #1 

**What is the new behavior (if this is a feature change)?**
- Now there is a way to make a quick feature request. Closes #1 

**Other information**:
- Ref: https://en.wikipedia.org/wiki/Goldfish#/media/File:Katri.jpg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,7 @@
 
 The first attempt to resurrect ProjectSky w/ the power of git, react and mdbreact.
 
+f u goldfish
```
Comment (petschna) [inline README.md]: Please add a comment to explain the functionality of this fish

**Review by petschna — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by petute — APPROVED** [review]
Should be ok to merge.

**Review by petschna — APPROVED** [review]
Should be ok to merge

**Review by kleberbaum — COMMENTED** [review]
should be ok to merge.

**Pull request merged**